app/main.py

```diff
@@ -1,13 +1,24 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {"line": error["line_number"], "column": error["column_number"],
+            "message": error["text"], "name": error["code"],
+            "source": "flake8"}
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+
+    return {"errors": [{"line": item["line_number"],
+                        "column": item["column_number"],
+                        "message": item["text"], "name": item["code"],
+                        "source": "flake8"}
+                       for item in errors], "path": file_path,
```

Comment:
Use format_linter_error to avoid code duplication

Reply:
already done